dali/operators/reader/numpy_reader_op.cc

```diff
@@ -240,7 +240,7 @@ void NumpyReaderCPU::RunImpl(HostWorkspace &ws) {
 
   // From 1 to 10 blocks per sample depending on the nthreads/nsamples ratio
   int blocks_per_sample = std::max(1, 10 * nthreads / nsamples);
-  constexpr int kThreshold = 16000;  // smaller samples will not be subdivided
+  constexpr int kThreshold = 16 << 10;  // smaller samples will not be subdivided
```

Comment:
This constant appears in a few places. Does it make sense to name it and put it in some common place?

Reply:
Makes sense. Will do